server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/ObjectStorageDependenciesModule.java

```diff
@@ -96,4 +91,7 @@ private ObjectStorageBlobsDAO buildObjectStore(ObjectStorageBlobConfiguration co
         }
     }
 
+    private PayloadCodec buildPayloadCodec(ObjectStorageBlobConfiguration configuration) {
+        return configuration.getPayloadCodecFactory().create(configuration);
```

Comment:
This line seems boiler plate.

Can't we have a simple `configuration.getPayloadCodec()` ?